main.py

```diff
@@ -615,7 +615,7 @@ def get_redirect_route(
     get_redirect_route(r'/userinfohandler', profile.UserInfoHandler),
     get_redirect_route(r'/userinfohandler/data', profile.UserInfoHandler),
     get_redirect_route(r'/url_handler', profile.UrlHandler),
-    get_redirect_route(r'/moderator', moderator.ModeratorPage),
+    get_redirect_route(r'/moderator', oppia_root.OppiaRootPage),
```

Comment:
This is not needed i.e, you can remove this line as all the routes in `PAGES_REGISTERED_WITH_FRONTEND` are [already registered around line 1145](https://github.com/oppia/oppia/blob/d149ea60ad0c734898c1c539c58553909e0f66fe/main.py#L1145)

Reply:
Done